### PR TITLE
alternative for #144

### PR DESCRIPTION
**Pull request by ocefpaf**
Not sure why #144 is failing so let's try it again...

Ping @sdvillal.

### CODE REVIEW

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -343,6 +345,8 @@ jasper:
   - 1.900.1
 jpeg:
   - 9
+libjpeg-turbo
```
Comment (jakirkham) [inline recipe/conda_build_config.yaml]: Think this needs to be an underscore like `json_c` below.

Comment (ocefpaf) [reply recipe/conda_build_config.yaml]: Why it is not the package name? If that is a yaml thing we should fix that and not workaround it by creating confusing `-` and `_` changes instead.

Comment (isuruf) [reply recipe/conda_build_config.yaml]: It's a conda-build thing. these variables are used as jinja variables and env variables and therefore `-` is not allowed.

Comment (ocefpaf) [reply recipe/conda_build_config.yaml]: :-(

Thanks for the info @isuruf!

File: recipe/conda_build_config.yaml [hunk 2]
```diff
@@ -316,7 +318,7 @@ gst_plugins_base:
 gdal:
   - 2.3
 geos:
-  - 3.6.2
+  - 3.7.0
```
Comment (ocefpaf) [inline recipe/conda_build_config.yaml]: @pelson and @dopplershift how is cartopy with `geos 3.7.0`?

Comment (dopplershift) [reply recipe/conda_build_config.yaml]: Looks like our test suite runs fine with geos 3.7--so long as Shapely is also built with it.

Comment (ocefpaf) [reply recipe/conda_build_config.yaml]: If `cartopy` is OK let's roll this change. (The central pinning and conda-build 3 should ensure that no mismatch of shapely/cartopy/geos would happen.)

Comment (dopplershift) [reply recipe/conda_build_config.yaml]: So, by moving to geos 3.7, we've eliminated Python 2.7 builds on Windows...

Comment (dopplershift) [reply recipe/conda_build_config.yaml]: Could we make the pin conditional?

**Pull request merged**